Gemfile

```diff
@@ -21,13 +21,18 @@ gem 'jquery-rails'
 
 # Use datatables for rendering and searching songs
 gem 'jquery-datatables-rails', '~> 3.3.0'
+gem 'will_paginate'
 
+gem 'yomu'
+gem 'tesseract-ocr'
```

Comment:
The tesseract-ocr gem hasn't been updated to use the API breaking changes that the newest version of the Tesseract C lib uses. I had to point brew to an older version of the lib. This worked for me: http://stackoverflow.com/a/33636970